addon/components/mdc-toolbar/title.js

```diff
@@ -1,16 +1,21 @@
 import Ember from 'ember';
 import layout from '../../templates/components/mdc-toolbar/title';
-import { MDCToolbarFoundation } from '@material/toolbar';
-
-const { strings } = MDCToolbarFoundation;
 
 export default Ember.Component.extend({
   //region Ember Hooks
   layout,
-  classNames: [`${strings.TITLE_SELECTOR}`],
+  classNames: ['mdc-toolbar__title'],
+  classNameBindings: ['fixed:mdc-toolbar__title--fixed'],
```

Comment:
In [the docs](https://material.io/components/web/catalog/toolbar/) I didn't see anything about the class `mdc-toolbar__title--fixed` -- only about `mdc-toolbar__title` and `mdc-toolbar--fixed`.

Reply:
Good catch!